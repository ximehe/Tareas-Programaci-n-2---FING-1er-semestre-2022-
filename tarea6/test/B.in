invertirPrioridad
insertarEnCP 1 2
invertirPrioridad
estaEnCP 1
prioritario
insertarEnCP 2 10
insertarEnCP 3 9
insertarEnCP 4 8
insertarEnCP 5 7
prioritario
invertirPrioridad
prioritario
Fin