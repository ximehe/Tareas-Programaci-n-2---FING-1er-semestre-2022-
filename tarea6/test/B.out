1>Se cambi칩 el criterio de prioridad.
2>Se intenta insertar 1 con prioridad 2.00.
3>Se cambi칩 el criterio de prioridad.
4>En la cp est치 1.
5>El prioritario es 1
6>Se intenta insertar 2 con prioridad 10.00.
7>Se intenta insertar 3 con prioridad 9.00.
8>Se intenta insertar 4 con prioridad 8.00.
9>Se intenta insertar 5 con prioridad 7.00.
10>El prioritario es 1
11>Se cambi칩 el criterio de prioridad.
12>El prioritario es 2
13>Fin.
